pkg/controller/plan/kubevirt.go

```diff
@@ -797,7 +799,12 @@ func (r *KubeVirt) getStorageProfileModes(storageName string) (accessModes []cor
 
 // Return true when the import is done with OvirtVolumePopulator
 func (r *KubeVirt) useOvirtPopulator(vm *plan.VMStatus) bool {
-	return *r.Plan.Provider.Source.Spec.Type == v1beta1.OVirt && vm.Warm == nil && r.Destination.Provider.IsHost()
+	return r.isOvirt() && vm.Warm == nil && r.Destination.Provider.IsHost()
+}
+
+// Return true when the import is done from oVirt source.
+func (r *KubeVirt) isOvirt() bool {
```

Comment:
it's a bit awkward to have a method named `isOvirt` on `KubeVirt`.. it can be named `isImportFromOvirt` or something like that but if we just want to make it shorter, how about replacing `*r.Plan.Provider.Source.Spec.Type` with `r.Plan.Provider.Source.Type()` ?

Reply:
i just returned it to the way it was.